### PR TITLE
Added WiFi docs

### PR DESCRIPTION
**Pull request by fahadboss10**
### Description
Added wifi component images in assets and added docs.

### Checklist:
- [x] I have proofread the changes. (Spellings, grammar, etc)
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by fahadboss10** [comment]
> 
> 
> The style of this doc doesn't match the docs please refer the screenshot.md under utilities

What is needed?

File: docs/components/connectivity/wifi.md [hunk 1]
```diff
@@ -1,4 +1,86 @@
...
+
+## Overview
+
+The **WiFi** component can be used to manage the WiFi settings as well as to retrieve information about device's WiFi.
+
+
+
+## Methods
+
+### BSSID
```
[the remark this answers is not in the document]
Comment (fahadboss10) [reply docs/components/connectivity/wifi.md]: Thank you for guiding me! 
One more question: you mean like bssid is getter only? And type should be text?

[the remark this answers is not in the document]
Comment (fahadboss10) [reply docs/components/connectivity/wifi.md]: I have edited as you said. Now you can check.

**Comment by fahadboss10** [comment]
I'm waiting for review about this doc I created so that if it is OK I can make others. 
One week has already past.

File: docs/components/connectivity/wifi.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SivagiriVisakan) [reply docs/components/connectivity/wifi.md]: Hi @fahadboss10 !
The `[ Getter only ]` text is set only on _component properties_.
It is not needed for _methods_.

[the remark this answers is not in the document]
Comment (fahadboss10) [reply docs/components/connectivity/wifi.md]: > The `[ Getter only ]` text is set only on _component properties_.
> It is not needed for _methods_.

Ok, *Disable* and *Enable* are methods. Other like *BSSID* and *IP Address* are also considered methods or not?

[the remark this answers is not in the document]
Comment (SivagiriVisakan) [reply docs/components/connectivity/wifi.md]: Yes, **BSSID** and **IP Address** are also methods.
They are methods with a return value while **Disable** and **Enable** are methods _without_ return value.

[the remark this answers is not in the document]
Comment (fahadboss10) [reply docs/components/connectivity/wifi.md]: I've edited. Can you check now, please?

File: docs/components/connectivity/wifi.md [hunk 2]
```diff
@@ -1,4 +1,103 @@
...
+## Overview
+
+The **WiFi** component can be used to manage the WiFi settings as well as to retrieve information about device's WiFi.
+
+## Methods
+
+### BSSID
+![](/assets/images/components/connectivity/wifi/m_bssid.png)
+
+\[None\]
```
Comment (Vishwas-Adiga) [inline docs/components/connectivity/wifi.md]: Should be \[ None \]
(notice the spaces)

File: docs/components/connectivity/wifi.md [hunk 3]
```diff
@@ -1,4 +1,103 @@
...
+
+Returns the basic service set identifier (BSSID) of the current access point.
+
+### Disable
+![](/assets/images/components/connectivity/wifi/m_disable.png)
+
+\[ None \]  
+
+Disable the Wi-Fi
```
Comment (Vishwas-Adiga) [inline docs/components/connectivity/wifi.md]: Method descriptions should be in the format: "This method [description]", where the text "This method" is omitted.
In this case, the desc should be "This method **Disables the WiFi**"

File: docs/components/connectivity/wifi.md [hunk 4]
```diff
@@ -1,4 +1,103 @@
...
+\[ None \]
+
+Enable the Wi-Fi
+
+### Is 5GHz Supported
+![](/assets/images/components/connectivity/wifi/m_is-5ghz-supported.png)
+
+\[ None \]
+
+Return whether this adapter supports 5GHz band.
```
Comment (Vishwas-Adiga) [inline docs/components/connectivity/wifi.md]: Should be "Returns ..." (see above)

Also, how about providing some more info about the adapter and 5GHz band? You can place additional information in the `!!! info` dialog (see https://docs.kodular.io/components/utilities/color-utilities/#convert-int-to-hex)

File: docs/components/connectivity/wifi.md [hunk 5]
```diff
@@ -1,4 +1,103 @@
...
+* `False` - 5GHz band is not supported by the adapter.
+
+### Is Enabled
+![](/assets/images/components/connectivity/wifi/m_is-enabled.png)
+
+\[ None \]
+
+Return whether Wi-Fi is enabled or disabled.
+
+* `True` - WiFi is enabled.
```
Comment (Vishwas-Adiga) [inline docs/components/connectivity/wifi.md]: True/False methods should follow this pattern:
https://docs.kodular.io/components/utilities/color-utilities/#is-dark-color

File: docs/components/connectivity/wifi.md [hunk 6]
```diff
@@ -1,4 +1,103 @@
...
+* `False` - WiFi is disabled.
+
+### Is WiFi Direct Supported
+![](/assets/images/components/connectivity/wifi/m_is-wifi-direct-supported.png)
+
+\[ None \]
+
+Return whether this adapter supports Wi-Fi Direct.
```
Comment (Vishwas-Adiga) [inline docs/components/connectivity/wifi.md]: Maybe provide some more info on WiFi direct?

File: docs/components/connectivity/wifi.md [hunk 7]
```diff
@@ -1,4 +1,103 @@
...
+Returns the received signal strength indicator of the current 802.11 network, in dBm.
+
+### Toggle
+![](/assets/images/components/connectivity/wifi/m_toggle.png)
+
+\[ None \]
+
+Toggles the WiFi.
+
+- If WiFi is on, it will be set to off
```
Comment (Vishwas-Adiga) [inline docs/components/connectivity/wifi.md]: See comment above on True/False methods

File: docs/components/connectivity/wifi.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Vishwas-Adiga) [reply docs/components/connectivity/wifi.md]: LGTM 👍

**Pull request closed without merging**